test/Spec/State/ForPairs.spec.luau

```diff
@@ -33,7 +33,7 @@ return function()
 		local expect = getfenv().expect
 		
 		local scope = {}
-		local forObject = ForPairs(scope, {}, function()
+		local forObject = ForPairs(scope, {}, function(use, scope, key, value)
 			-- intentionally blank
```

Comment:
Use, Scope, Key, and Value shouldn't be provided as it is never used

Reply:
reverted